vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -109,6 +135,32 @@ def parse_pod(pod)
       new_result
     end
 
+    def parse_endpoint(entity)
+      new_result = parse_base_item(entity)
+      new_result[:container_groups] = []
+
+      # remove once kubernetes api v1beta3/v1.0 will be stable, see:
+      # https://github.com/GoogleCloudPlatform/kubernetes/commit/8ae20382
+      if !entity.respond_to?(:subsets) || entity.subsets.nil?
+        $log.warn("your kubernetes endpoints are not providing subsets, " \
```

Comment:
Ruby/rubocop disagree:

    kubernetes.rb:146:19: E: unexpected token tSTRING
                  "please upgrade (see commit 8ae20382")

maybe you intended that it should be replaced by something else? ...e.g. "+" or ","? in that case adding/concatenating strings seems less efficient.